src/data/trainers/trainer-config.ts

```diff
@@ -1242,14 +1242,49 @@ export const trainerConfigs: TrainerConfigs = {
     .setEncounterBgm(TrainerType.POKEFAN)
     .setHasGenders("Breeder Female")
     .setHasDouble("Breeders")
-    .setPartyTemplateFunc(() =>
-      getWavePartyTemplate(
-        trainerPartyTemplates.FOUR_WEAKER,
-        trainerPartyTemplates.FIVE_WEAKER,
-        trainerPartyTemplates.SIX_WEAKER,
-      ),
-    )
-    .setSpeciesFilter(s => s.baseTotal < 450),
+    .setPartyTemplates(trainerPartyTemplates.FOUR_WEAK, trainerPartyTemplates.FIVE_WEAK)
+    .setSpeciesPools({
+      [TrainerPoolTier.COMMON]: [
+        SpeciesId.PICHU,
+        SpeciesId.CLEFFA,
+        SpeciesId.IGGLYBUFF,
+        SpeciesId.TOGEPI,
+        SpeciesId.TYROGUE,
+        SpeciesId.SMOOCHUM,
+        SpeciesId.ELEKID,
+        SpeciesId.MAGBY,
+        SpeciesId.AZURILL,
+        SpeciesId.WYNAUT,
+        SpeciesId.BUDEW,
+        SpeciesId.CHINGLING,
+        SpeciesId.BONSLY,
+        SpeciesId.MIME_JR,
+        SpeciesId.HAPPINY,
+        SpeciesId.MANTYKE,
+        SpeciesId.TOXEL,
+      ],
+      [TrainerPoolTier.UNCOMMON]: [SpeciesId.DITTO, SpeciesId.MUNCHLAX, SpeciesId.RIOLU, SpeciesId.AUDINO],
```

Comment:
```suggestion
        SpeciesId.AZURILL,
        SpeciesId.WYNAUT,
        SpeciesId.BUDEW,
        SpeciesId.CHINGLING,
        SpeciesId.BONSLY,
        SpeciesId.MIME_JR,
        SpeciesId.HAPPINY,
        SpeciesId.MANTYKE,
        SpeciesId.TOXEL,
      ],
      [TrainerPoolTier.UNCOMMON]: [
        SpeciesId.DITTO,
        SpeciesId.ELEKID,
        SpeciesId.MAGBY,
        SpeciesId.MUNCHLAX,
        SpeciesId.RIOLU,
        SpeciesId.AUDINO,
      ],
```

I think Electivire and Magmortar are sufficiently powerful that they can be put in Uncommon, since they're now guaranteed fully evolved after 80